nixos/lib/testing/run.nix

```
@@ -43,10 +71,15 @@ in
   };
 
   config = {
-    rawTestDerivation =
+    rawTestDerivation = hostPkgs.stdenv.mkDerivation (
+      # TODO: make mkDerivation accept a trivial functor as a function
+      a: config.rawTestDerivationArg a
```

After https://github.com/NixOS/nixpkgs/pull/411760,

```suggestion
      onfig.rawTestDerivationArg
```